setup.py

```diff
@@ -19,6 +19,7 @@
       packages=find_packages(exclude=['ez_setup', 'examples', 'tests']),
       include_package_data=True,
       zip_safe=False,
+      requires= 'python_magic',
```

Comment:
Ah! I think you made it right and the following lines a wrong. :) Would you mind to put the following packages under "install_requires" into the requires and delete "install_requires"?